### ISSUE TITLE
Add AddColumn in AlterTableOperation.

### ISSUE THREAD

**Issue by ivanceras** [andygrove/sqlparser-rs#147]
The AlterTable statements seems to lack support for adding a new column to an existing table.

```rust
pub enum AlterTableOperation {
    AddConstraint(TableConstraint),
    DropConstraint { name: Ident },
}
```

**Comment by ivanceras** [comment]
@andygrove you can add me as a collaborator/maintainer of the project. I can help with code reviews and releases.

**Comment by nickolay** [comment]
This is implemented in #203 and #148. Thanks!
